README.md

```diff
@@ -28,31 +28,134 @@ flag is highly recommended for those new to [pip](https://pip.pypa.io).
 A PEX executable (Windows not supported) and source releases are
 [here](https://github.com/planetlabs/planet-client-python/releases/latest).
 
-## Documentation
+## API Key
 
-Online documentation:
-https://planetlabs.github.io/planet-client-python/index.html
+The planet API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
 
-Documentation is also provided for download
-[here](https://github.com/planetlabs/planet-client-python/releases/latest).
+This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
 
+Using `planet init` your account credentials (login/password) can be used to obtain the api key.
 
-## Development
+## Quick Start
 
-To contribute or develop with this library, see
-[CONTRIBUTING](https://github.com/planetlabs/planet-client-python/CONTRIBUTING.md)
+### Python Library
 
+The python library is asynchronous, which greatly speeds up many interactions
+with the planet API. 
 
-## API Key
+```python
+import asyncio
+import os
+from planet.api import APlanetSession, AOrdersClient
+from planet.api.order_details import OrderDetails, Product
 
-The API requires an account for use. [Signup here](https://www.planet.com/explorer/?signup).
+API_KEY = os.getenv('PL_API_KEY')
 
-This can be provided via the environment variable `PL_API_KEY` or the flag `-k` or `--api-key`.
+image_ids = ['3949357_1454705_2020-12-01_241c']
+order_detail = OrderDetails(
+    'test_order',
+    [Product(image_ids, 'analytic', 'psorthotile')]
+)
 
-Using `planet init` your account credentials (login/password) can be used to obtain the api key.
+async def create_order(order_detail):
+    async with APlanetSession(auth=(API_KEY, '')) as ps:
+        client = AOrdersClient(ps)
+        return await client.create_order(order_detail)
 
+oid = asyncio.run(create_order())
+print(oid)
+```
+
+Not into async? No problem. Just wrap the library and async operations together
+and call from your synchronous code.
 
-# Example CLI Usage
+```python
+def sync_create_order(order_details):
```

Comment:
I hear you. I agree, this would be nice. Here's the deal with async and sync - really, if we want to support sync in our library, we should reimplement all the async commands as sync (and they shouldn't be each running an event loop - why? right now all I can say is it's frowned upon, maybe it is costly to spin up an event loop and will slow things down?). Implementing sync versions of every public function (and the lower level functionality as well) is a lot of redundancy to take on. I now remember that I was going to add an issue to track requests for synchronous support within the library and then link to it in the README so users can weigh in and we can assess when/if it makes sense to add synchronous support.

Added #251 and updated README to include more information about usage of `asyncio.run` and a link to the ticket.

Does that feel adequate for now?

Reply:
Marking this as resolved. Feel free to follow up either inside that ticket or with a new ticket for continued discussion.